Leto-Tasks.csv

```diff
@@ -0,0 +1,7 @@
+D,N,20,2021-03-02,,
```

Comment:
Not sure if you want to commit the data file as well. 
Maybe could specify it as a default pre-loaded list.

Reply:
I think this was committed accidentally should be in the git ignore in the future I think!